lib/bitcoin/BitcoinClient.ts

```diff
@@ -939,6 +939,20 @@ export default class BitcoinClient {
           continue;
         }
 
+        // Error -28 - Client is still warming up
+        try {
+          const responseBody = JSON.parse(bodyBuffer.toString());
+          if (responseBody.error?.code === -28) {

```

Comment:
Assuming Bitcoin Core is initializing as the test simulates, then adding retry here doesn't help with anything right? Since it can easily take minutes verify blocks. Am I misunderstanding something?